client/packages/sdk/src/circuit/tx.ts

```diff
@@ -32,6 +24,51 @@ export class Tx {
     this.exportMode = exportMode;
   }
 
+  /**
+   *
+   * @param tx - The transaction to send
+   * @param options - Options for transaction
+   *
+   * @returns Promise
```

Comment:
Just a suggestion:
```suggestion
   * @param {SubmittableExtrinsic}       tx - The transaction to send
   * @param {Partial<SignerOptions>}  options - Options for transaction
   *
   * @returns {Promise<string>}
```

This will improve the intellisense suggestions and add pop-out docs (like in Rust) when looking up the function in the IDE.

Reply:
Ok, jsdoc installed